<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackagesPage</class>
 <widget class="QWidget" name="PackagesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="6">
    <widget class="QTableView" name="_listPackages">
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="_buttonAdd">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add a new package to grauchy.&lt;/p&gt;&lt;p&gt;Shortcut: &lt;span style=&quot; font-weight:600;&quot;&gt;Ins&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>&amp;Add</string>
     </property>
     <property name="icon">
      <iconset resource="grauchy.qrc">
       <normaloff>:/resources/edit_add.png</normaloff>:/resources/edit_add.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="_buttonEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Edit current package and its hotkeys.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>&amp;Edit</string>
     </property>
     <property name="icon">
      <iconset resource="grauchy.qrc">
       <normaloff>:/resources/edit_view.png</normaloff>:/resources/edit_view.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="_buttonRemove">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remove current package and all its hotkeys.&lt;/p&gt;&lt;p&gt;Shortcut: &lt;span style=&quot; font-weight:600;&quot;&gt;Del&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>&amp;Remove</string>
     </property>
     <property name="icon">
      <iconset resource="grauchy.qrc">
       <normaloff>:/resources/edit_remove.png</normaloff>:/resources/edit_remove.png</iconset>
     </property>
     <property name="shortcut">
      <string>Del</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>227</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="_buttonExport">
     <property name="text">
      <string>E&amp;xport...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="_buttonImport">
     <property name="text">
      <string>&amp;Import...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="grauchy.qrc"/>
 </resources>
 <connections/>
</ui>
